inc/namespace.php

```diff
@@ -26,6 +29,9 @@ function bootstrap() {
  */
 function load_notifications() {
 	Notifications\setup();
+
+	add_filter( 'qm/collectors', __NAMESPACE__ . '\\register_workflow_notification_qm_collector' );
```

Comment:
Soz, last thing to do here - I think these filters and the callbacks should go into the `inc/notifications/namespace.php` file so it's all contained there. That way you won't need the first 2 `use` statements added here as you'll already be in the right namespace.